src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+// eslint-disable-next-line
+import Calculator from './components/calculator';
```

Comment:
- [ ] The app is not rendering the Calculator.js component. This is because you have not imported it correctly. Kindly check and fix it, please.